examples/GE-Signa-PETMR/template.hs

```diff
@@ -0,0 +1,49 @@
+!INTERFILE  :=
+!imaging modality := PT
+name of data file := template.s
+originating system := GE PET/MR Signa
+!version of keys := STIR4.0
+!GENERAL DATA :=
+!GENERAL IMAGE DATA :=
+!type of data := PET
+imagedata byte order := LITTLEENDIAN
+!PET STUDY (General) :=
+!PET data type := Emission
+applied corrections := {None}
+!number format := float
+!number of bytes per pixel := 4
+number of dimensions := 4
+matrix axis label [4] := segment
+!matrix size [4] := 45
+matrix axis label [3] := view
+!matrix size [3] := 224
+matrix axis label [2] := axial coordinate
+!matrix size [2] := { 1,5,9,13,17,21,25,29,33,37,41,45,49,53,57,61,65,69,73,77,81,85,89,85,81,77,73,69,65,61,57,53,49,45,41,37,33,29,25,21,17,13,9,5,1}
+matrix axis label [1] := tangential coordinate
+!matrix size [1] := 357
+minimum ring difference per segment := { -44,-43,-41,-39,-37,-35,-33,-31,-29,-27,-25,-23,-21,-19,-17,-15,-13,-11,-9,-7,-5,-3,-1,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44}
+maximum ring difference per segment := { -44,-42,-40,-38,-36,-34,-32,-30,-28,-26,-24,-22,-20,-18,-16,-14,-12,-10,-8,-6,-4,-2,1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43,44}
+Scanner parameters:= 
+Scanner type := GE PET/MR Signa
+Number of rings                          := 45
+Number of detectors per ring             := 448
+Inner ring diameter (cm)                 := 62.38
```

Comment:
`list_projdata_info template.hs` says:

WARNING: Interfile warning: 'inner ring diameter (cm)' (62.380000) is expected to be 62.359998.